smoke-test/src/main/java/org/opennms/smoketest/selenium/AbstractOpenNMSSeleniumHelper.java

```diff
@@ -206,6 +206,11 @@ protected void failed(final Throwable e, final Description description) {
         @Override
         protected void finished(final Description description) {
             cleanUp();
+
+            // Ensure we close the driver, otherwise the browser is still open
+            // which might cause the container to run out of memory when running either
+            // in debug mode or a parameterized test
+            getDriver().quit();
```

Comment:
Looks like this may be causing some failures now:
```
org.openqa.selenium.NoSuchSessionException: 
Session ID is null. Using WebDriver after calling quit()?
```

Reply:
Yep, I saw this, will take a look. Adding the `triage` label though, as it may take a while.